reana-ui/src/pages/workflowList/components/WorkflowList.js

```diff
@@ -65,6 +66,7 @@ export default function WorkflowList() {
     if (!interval.current) {
       interval.current = setInterval(() => {
         getWorkflows();
+        setRefreshedAt(moment().format("LTS"));
```

Comment:
Usually I prefer 24/24 format, which does not need `PM` at the end, which gives us an option to display time zone?

``17:20:49 UTC`` instead of ``5:20:49 PM``